git/.gitignore

```diff
@@ -0,0 +1,130 @@
+# Logs
+logs
+*.log
+npm-debug.log*
```

Comment:
you have a git ignore but you pushed everything that you shouldn't have pushed xD